### PR TITLE
Update AmazonMetadataBuilder.php

### PR DESCRIPTION
**Pull request by bertrandserres**
Problem uploading to S3
https://github.com/sonata-project/SonataMediaBundle/issues/1160

I am targetting this branch, because {reason}.

Closes #{put_issue_number_here}

## Changelog

```markdown
### Added
- Added some `Class::newMethod` to do great stuff

### Changed

### Deprecated

### Removed

### Fixed

### Security
```

## To do

- [ ] Update the tests
- [ ] Update the documentation
- [ ] Add an upgrade note

### CODE REVIEW

File: Metadata/AmazonMetadataBuilder.php [hunk 1]
```diff
@@ -115,7 +115,10 @@ protected function getDefaultMetadata()
     protected function getContentType($filename)
     {
         $extension = pathinfo($filename, PATHINFO_EXTENSION);
-        $contentType = Mimetypes::getInstance()->fromExtension($extension);
+/* Code that fixed it. */
```
Comment (greg0ire) [inline Metadata/AmazonMetadataBuilder.php]: Please remove this comments

**Comment by bertrandserres** [comment]
I'm not a contributor of the project, I'm not aware of your PR guidelines, and I don't have write access to the branch.
I just want this line to be merged so that I can stop editing this file in my vendors directory.
Can you edit the file and PR comment on your side please ?

**Comment by greg0ire** [comment]
Please fork the repository and do it yourself.

**Comment by bertrandserres** [comment]
Really helpful... 
Thanks

**Comment by greg0ire** [comment]
I'm not your slave, sorry.

**Comment by bertrandserres** [comment]
Alright. I guess it was a tough week for you.
I'm gonna study all your PR process and come back later.
Many thanks for your help

**Comment by greg0ire** [comment]
Thanks for your understanding. I'll help you if your stuck at any point.

**Pull request closed without merging**